### ISSUE TITLE
What is the purpose of second anonymous column on 'My Spaces' page?

### ISSUE THREAD

**Issue by dgolovin** [openshiftio/openshift.io#2609]
`My Space`' page looks like:
![image](https://user-images.githubusercontent.com/620330/37437456-97a782e6-27aa-11e8-939a-1bfd7ba5531f.png)

As you can see second column has no name and always show my avatar/email. I am already on 'My Spaces' page and every line basically repeating my info.

**Comment by alexeykazakov** [comment]
It's not your email. It's your username (which happens to be the same as your email). But in general I agree. It looks a bit weird.

**Comment by joshuawilson** [comment]
@catrobson do you have an answer for this?

**Comment by catrobson** [comment]
We will be showing spaces that you own, as well as spaces that you collaborate in, which would have a different username next to it, which will hopefully help with why this information is shown.  This is also meant to scale for when we have organizations associated with spaces so we can show the organization name.  @dlabrecq can you validate for me whether we are currently showing spaces that I am a collaborator in within this list?

**Comment by dlabrecq** [comment]
That is the intention; however, we don't have an API that shows the spaces a user is a collaborator in. Right now, it only shows the user's spaces.

**Comment by catrobson** [comment]
Thanks @dlabrecq - @joshuawilson I think some of the issues Dan created around this got closed.  Is this feasible to add a story around this API to the next sprint, or at least get it on the backlog?

**Comment by dgolovin** [comment]
@catrobson, @dlabrecq, thank you for explanation 👍 So 'My Spaces' meaning is 'My Spaces I am collaborating in' and the second column could be named 'Owner', where 'Owner' value could be Person or Organization.

**Comment by joshuawilson** [comment]
yes, but since we are not collaborating much you will mostly just see you username there

**Comment by catrobson** [comment]
API needed for which spaces a user is a collaborator on issue:  https://github.com/openshiftio/openshift.io/issues/1630
